keps/sig-apps/20200529-delete-priority-annotations.md

```diff
@@ -0,0 +1,293 @@
+---
+title: Add delete priority annotation for ReplicaSet
```

Comment:
is this only for replicaset or for all pod-deleting controllers?

Reply:
That is a good question, as this is my first code contribution to the Kubernetes project, actually I don't know. Can you help me to find that out?

Reply:
It's not applicable for other controllers like StatefulSet or DaemonSet. They deterministically deletes Pods when scaling down.

Reply:
it is applicable to jobs when reducing parallelism.

Reply:
we can start with ReplicaSet though and add support for Jobs later.